### PR TITLE
CI/security: Replace snyk scannning with trivy scanning

### PR DESCRIPTION
**Pull request by sldblog**
## What does this pull request do?

Replace snyk scannning with trivy scanning

🔐 To make this pass, pinned `validator` to mitigate CVE-2021-3765

As an added bonus, now with Slack alerts if nightly scan fails :)

## What is the intent behind these changes?

Snyk has been replaced with Trivy as it's faster, smaller, and better
integrated.

### CODE REVIEW

**Comment by sldblog** [comment]
📝 This will **fail** with good reason

image

File: package.json [hunk 1]
```diff
@@ -91,7 +91,7 @@
       "prettier --write"
     ]
   },
-  "//": "tar is not a direct dependency of our project; we’ve only added it here to remove a security vulnerability introduced by a transitive dependency. This should be removed; see IC-2177",
+  "//": "validator is pinned for CVE-2021-3765; remove when fixed upstream",
```
Comment (jacksonj04) [inline package.json:94]: Is it worth making this a `TODO` so that it shows up when searching for things we need to look at?

**Review by jacksonj04 — APPROVED** [review]
One comment about making sure we pick up unpinning validator in the future, otherwise 🚢

File: package.json (hunk 1 above)
Comment (sldblog) [reply package.json]: 👍 added in c2562d1500f2bf1fdd99819c1c24925038e7b39c

**Pull request merged**